runtime/lib/cloudapi/index.js

```diff
@@ -72,18 +74,20 @@ CloudStore.prototype.connect = function connect (masterId) {
         if (!res.success) {
           throw new Error(`cloud login failed ${res.msg}(${res.code})`)
         }
-        handleResponse(JSON.parse(res.data))
-        return resolve(true)
+        return resolve(JSON.parse(res.data))
       } catch (err) {
         logger.error(err)
         return reject(err)
       }
     }
-  }).then(() => {
+  }).then((data) => {
+    return this.handleResponse(data)
+  }).then((data) => {
+    logger.info(`handle response ${data}`)
```

Comment:
Seems unnecessary log and unaccessible data.

Reply:
removed